lib/src/api.dart

```diff
@@ -360,70 +366,89 @@ class NDEFRecord {
   /// null.
   final String languageCode;
 
+  /// rawPayload contains the raw payload provided by the reader.
+  /// It will only be set when reading NDEF tags. Otherwise it will be null.
```

Comment:
The `NDEFRecord` wouldn't exist if the tag wasn't NDEF formatted so this comment is pointless.

Reply:
What I meant was that the field is only set when reading a tag, but not when instantiating a new record with one of the constructors.

Reply:
Oh yeah.
I totally forgot writing was a thing. I don't do much NFC these days.
My bad!